### ISSUE TITLE
adapt to SE-0274: Use #filePath instead of #file on newer Swift versions

### ISSUE THREAD

**Issue by weissi** [apple/swift-nio-http2#188]
This SE I think will be breaking the test suite: https://forums.swift.org/t/accepted-se-0274-concise-magic-file-names/34115

**Comment by weissi** [comment]
I think the fix is

```swift
#if compiler(>=5.3)
.....(#filePath)
#else
.....(#file)
#endif
```

We need to confirm which Swift version this will go into. I'm pretty sure this won't go into Swift 5.2 but it seems clear that it'll go into whatever the next Swift version is called. Both 6.0 and 5.3 will be hit by `compiler(>=5.3)`.

Removing good-first-issue for now until we've got clarity what the first version to make this breaking change will be though.

**Comment by Lukasa** [comment]
I believe the advice on the Swift forums was actually to use the Bundle API going forward.

**Comment by Lukasa** [comment]
https://forums.swift.org/t/concise-magic-file-names/31297/38

**Comment by Lukasa** [comment]
Resolved by #209.
